holoviews/core/util.py

```diff
@@ -647,6 +647,30 @@ def walk_depth_first(name):
                                     (names_by_level.get(i, None)
                                      for i in itertools.count())))
 
+
+def is_cyclic(graph):
+    """Return True if the directed graph g has a cycle."""
```

Comment:
What is the representation of the graph? A list of edges as tuples? Would be good to mention in the docstring.